src/config/process.ts

```diff
@@ -103,12 +105,14 @@ export const LND_HEALTH_REFRESH_TIME_MS = parseInt(
 )
 
 export const getLoopConfig = () => {
-  if (!process.env.LND1_LOOP_TLS) throw new ConfigError("Missing LND1_LOOP_TLS config")
-  if (!process.env.LND2_LOOP_TLS) throw new ConfigError("Missing LND2_LOOP_TLS config")
-  if (!process.env.LND1_LOOP_MACAROON)
-    throw new ConfigError("Missing LND1_LOOP_MACAROON config")
-  if (!process.env.LND2_LOOP_MACAROON)
-    throw new ConfigError("Missing LND2_LOOP_MACAROON config")
+  if (getCronConfig().swapEnabled) {
```

Comment:
> this should be part of service validation
copied from https://github.com/GaloyMoney/galoy/pull/1733#discussion_r1007103937

Reply:
i did not think we were allowed to call process.env from any other file other than `process.ts`?

Reply:
I think we are beck to the original problem with https://github.com/GaloyMoney/galoy/commit/eab6cd41a50e4af67934fbd277726dff945fad85
since the process.ts is called every time and will throw the error when `getLoopConfig` is called.